auth.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const jwt = require('express-jwt');
+const jwksRsa = require('jwks-rsa');
+const jwtAuthz = require('express-jwt-authz');
```

Comment:
still using this?

Reply:
yes

Reply:
this is for scopes, since we arent using those, no need.